sympy/stats/tests/test_continuous_rv.py

```diff
@@ -31,7 +31,7 @@
 
 x, y, z = map(Symbol, 'xyz')
 
-
+@XFAIL
 def test_single_normal():
```

Comment:
Why is this XFAIL?

Reply:
This test is failing. I am not familiar with the code in the `stats` part, so I couldn't figure it out.
The result I am getting is:
```py
>>> from sympy import *
>>> from sympy.stats import Normal, quantile
>>> x = Symbol('x')
>>> mu = Symbol('mu', real=True)
>>> sigma = Symbol('sigma', positive=True)
>>> X = Normal('x', 0, 1)
>>> Y = X*sigma + mu
>>> quantile(Y)(x)
{sqrt(2)*sigma*(sqrt(2)*mu/(2*sigma) + erfinv(2*x - 1))}

## or
⎧     ⎛√2⋅μ                  ⎞⎫
⎨√2⋅σ⋅⎜──── + erfinv(2⋅x - 1)⎟⎬
⎩     ⎝2⋅σ                   ⎠⎭

```
while the original result is:
```py
>>> Intersection(S.Reals, FiniteSet(sqrt(2)*sigma*(sqrt(2)*mu/(2*sigma) + erfinv(
2*x - 1))))
    ⎧     ⎛√2⋅μ                  ⎞⎫
ℝ ∩ ⎨√2⋅σ⋅⎜──── + erfinv(2⋅x - 1)⎟⎬
    ⎩     ⎝2⋅σ                   ⎠⎭
```

Reply:
We need to check the assumptions on that expression. The intersection will evaluate if the expression is deemed to be real (or not).

Reply:
The intersection doesn't evaluate, the operation `quantile(Y)(x)` is resulting the evaluated result only.

Reply:
the result you are getting is correct, may be the code of `quantile` needs some changes in assumptions of variables, try some debugging and check where the `Intersection` is generated.

Reply:
I would say your result looks more intuitive, change the test if needed.

Reply:
I'm unresolving this because the XFAIL is still there.

Reply:
XFAIL has been removed